runner/webserver.ts

```diff
@@ -75,6 +76,12 @@ export function webserver(wct: Context): void {
       } catch (e) {
         // Safely ignore.
       }
+      const packageName = getPackageName(options);
+      const isPackageScoped = packageName && packageName[0] === '@';
+      if (isPackageScoped) {
+        browserScript = `../${browserScript}`;
```

Comment:
It'd be nice if we just calculated a "path to packages root" and then reused it as a prefix to address other package. This is fine for now.